README.md

```diff
@@ -66,11 +67,9 @@ $ sidekick --health-path=/minio/health/ready --address :8000 http://minio{1...4}
 
 ### Two sites with 4 servers each
 ```
-$ sidekick --health-path=/minio/health/ready http://site1-minio{1...4}:9000 http://site2-minio{1...4}:9000
+$ sidekick --health-path=/minio/health/ready http://site1-minio{1...4}:9000,http://site2-minio{1...4}:9000
```

Comment:
```suggestion
$ sidekick --health-path=/minio/health/ready http://site1-minio{1...4}:9000 http://site2-minio{1...4}:9000
```

This is still two different sites @karlpokus

Reply:
You're right.